packages/devtools_app/lib/src/screens/performance/performance_screen.dart

```diff
@@ -380,12 +383,29 @@ class MoreDebuggingOptionsButton extends StatelessWidget {
         extensions.disableOpacityLayers,
         extensions.disablePhysicalShapeLayers,
       ],
-      overlayDescription: Text(
-        'When toggling on/off a rendering layer, you will need '
-        'to reproduce activity in your app to see the effects of the '
-        'debugging option. All layers are rendered by default - disabling a '
-        'layer may help you identify expensive operations in your app.',
-        style: Theme.of(context).subtleTextStyle,
+      overlayDescription: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            'After toggling a rendering layer on/off, '
+            'reproduce the activity in your app to see the effects.
+            ' All layers are rendered by default - disabling a '
+            'layer might help identify expensive operations in your app.',
+            style: Theme.of(context).subtleTextStyle,
+          ),
+          if (serviceManager.connectedApp!.isProfileBuildNow!) ...[
+            const SizedBox(height: denseSpacing),
+            RichText(
+              text: TextSpan(
+                text:
+                    'These debugging options aren't available in profile mode. '
+                    'To use them, run your app in debug mode.',
+                style:
+                    textStyle.copyWith(color: theme.colorScheme.errorTextColor),
```

Comment:
since we now use this text style twice (here and in enhance_tracing.dart), add an extension method `subtleErrorTextStyle` to `ThemeDataExtension` in theme.dart. Then use that in both places.